From 768ef26bb2258ee41ce7125be11aaad0dc2ecd97 Mon Sep 17 00:00:00 2001
From: Simeon Marijon <simeon.marijon@foss.st.com>
Date: Wed, 12 Jul 2023 09:59:02 +0200
Subject: [PATCH 0432/1141] ARM: configs: revert REBOOT MODE support in
 fragment-02-multiv7_addons

This reverts commit ec3bd363eda7bffaea96c8b5c994593b903a24cf.

We remove the reboot mode based on syscon as we will introduce NVMEM API
for access to the TAMP backup registers. The reboot mode will use
nvmem-reboot-mode instead of syscon-reboot-mode. So we remove
SYSCON_REBOOT_MODE activation and it will be replace by
NVMEM_REBOOT_MODE.

Change-Id: Ia665c1027ea9e40c755cbbf06015686076eca1fb
Signed-off-by: Simeon Marijon <simeon.marijon@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 2 --
 1 file changed, 2 deletions(-)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -58,7 +58,6 @@ CONFIG_POWER_RESET_SYSCON=y
 CONFIG_PREEMPT=y
 CONFIG_PREEMPT_COUNT=y
 CONFIG_PROFILING=y
-CONFIG_REBOOT_MODE=y
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_REGULATOR_PROTECTION_CONSUMER=y
 CONFIG_REMOTEPROC=y
@@ -77,7 +76,6 @@ CONFIG_SERIAL_NONSTANDARD=y
 CONFIG_SRAM=y
 CONFIG_STM32_IPCC=y
 CONFIG_STM32_RPROC=y
-CONFIG_SYSCON_REBOOT_MODE=y
 CONFIG_TEE=y
 CONFIG_THERMAL=y
 CONFIG_TMPFS=y
